app/controllers/answers_controller.rb

```diff
@@ -1,2 +1,31 @@
 class AnswersController < ApplicationController
-end
+
+  before_action :logged_in_user
+
+  def new
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new
+    @lesson.category.words.each do |word|
+      unless @lesson.answers.where(word_id: word.id).exists?
+        @word = word 
+        break
+      end
+    end
+
+    if @word.nil?
+      redirect_to @lesson
+    end
+  end
+
+  def create
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = @lesson.answers.build(answer_params)
+    @answer.save
```

Comment:
@johnedelmapa
You can shorten these 2 lines by just using the `create` method
`@answer = @lesson.answers.create(answer_params)`